examples/createContract/src/spec.md

```diff
@@ -7,10 +7,12 @@ interface create_kid0()
 types
 
     KID : address Kid
+    Id  : uint256
 
 creates storage KID
 
-    #Kid.parent |-> ACCT_ID
+    #Kid.parent  |-> ACCT_ID
+    #Kid.ID      |-> Id
```

Comment:
I don't really understand this spec. Shouldn't there be something on the RHS of the storage conditions?

Reply:
Because the contract is being created this whole thing is actually on the RHS. There is no pre-state.

Reply:
This question made me realise there's an edge case here that needs addressing though I think. I've added a (currently failing) example with the storage of the new contract being modified via a subsequent function call.